tsconfig-common.json

```diff
@@ -26,6 +26,23 @@
 
         /* Completeness */
         "skipDefaultLibCheck": true, // it's safe to assume that built-in .d.ts files are correct
-        "skipLibCheck": false
+        // Setting `skipLibCheck: true`, avoids the following issues in various .d.ts files:
+        // - node_modules/@google-cloud/kms/node_modules/long/index.d.ts:446:1 - error
+        //   TS1203: Export assignment cannot be used when targeting ECMAScript modules.
+        //   Consider using 'export default' or another module format instead.
+        // - node_modules/@typescript-eslint/utils/node_modules/@typescript-eslint/typescript-estree/dist/ts-estree/ts-nodes.d.ts:19:3551 - error
+        //   TS2694: Namespace 'ts' has no exported member 'InputFiles'.
+        // - node_modules/@typescript-eslint/utils/node_modules/@typescript-eslint/typescript-estree/dist/ts-estree/ts-nodes.d.ts:19:3567 - error
+        //   TS2694: Namespace 'ts' has no exported member 'UnparsedSource'.
```

Comment:
`@typescript-eslint/typescript-estree` provides ambient type definitions that conflict with the TypeScript types for TypeScript itself.